en/django_installation/instructions.md

```diff
@@ -167,15 +167,15 @@ Before we do that, we should make sure we have the latest version of `pip`, the
 (myvenv) ~$ python3 -m pip install --upgrade pip
 ```
 
-Then run `pip install django~=1.11.0` (note that we use a tilde followed by an equal sign: `~=`) to install Django.
```

Comment:
I think it still makes sense to explicitly install a version so you're sure the right one is used. This is probably especially important for translations which could lag behind. OTOH 2.x should respect semver so chances of breakage is smaller.